charts/airflow/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: the community-maintained descendant of the stable/airflow helm chart - focused on ease-of-use
 name: airflow
-version: 8.1.3
+version: 8.1.4
```

Comment:
This change will need to be a minor patch (as we are adding a value)

In this case, we can do `8.3.0`